crates/ruff_python_ast/src/python_version.rs

```diff
@@ -59,6 +59,10 @@ impl PythonVersion {
         Self::PY313
     }
 
+    pub const fn latest_ty() -> Self {
+        Self::PY313
+    }
+
```

Comment:
I added this as a precaution that we don't accidentally bump the Python version in ty when bumping the version for ruff